packages/site/src/routes/[dictionaryId]/entries/print/PrintFieldCheckboxes.svelte

```diff
@@ -2,21 +2,23 @@
   import type { Readable } from 'svelte/store';
   import { t } from 'svelte-i18n';
   import { StandardPrintFields, type ExpandedEntry, type IPrintFields } from '@living-dictionaries/types';
+  import { convert_and_expand_entry } from '$lib/transformers/convert_and_expand_entry';
 
   export let entries: ExpandedEntry[];
   export let preferredPrintFields: Readable<IPrintFields>;
   export let showLabels: Readable<boolean>;
   export let showQrCode: Readable<boolean>;
 
   $: fieldsThatExist = Object.keys($preferredPrintFields).filter((field) => {
-    if (field === 'gloss') return true;
     return entries.find((entry) => {
-      if (field === 'alternateOrthographies')
-        return entry.local_orthography_1 || entry.local_orthography_2 || entry.local_orthography_3 || entry.local_orthography_4 || entry.local_orthography_5;
-      if (field === 'example_sentence') return entry.senses?.[0].example_sentences?.length;
-      if (field === 'sdn') return entry.senses?.[0].ld_semantic_domains_keys?.length;
-      if (field === 'image') return entry.senses?.[0].photo_files?.length;
-      if (field === 'speaker') return entry.sound_files?.[0].speakerName || entry.sound_files?.[0].speaker_ids?.length;
+      entry = convert_and_expand_entry(entry);
```

Comment:
Heed the types - this component's entries should already be expanded before coming in as you can see from the type. As you can see from https://github.com/livingtongues/living-dictionaries/blob/fix-print-view/packages/site/src/routes/%5BdictionaryId%5D/entries/print/%2Bpage.svelte#L127 it's the +page.svelte file's responsibility to provide expanded entries. Please update https://github.com/livingtongues/living-dictionaries/blob/fix-print-view/packages/site/src/routes/%5BdictionaryId%5D/entries/print/%2Bpage.svelte#L109 to `<PrintFieldCheckboxes entries={entries.map(convert_and_expand_entry)} {preferredPrintFields} {showLabels} {showQrCode} />`

Reply:
You're rigth! Sorry! I misundertood the problem